core/node/config-template.yaml

```diff
@@ -103,4 +103,6 @@ network:
 # Debug feature flags.
 disableBaseChain: <DISABLE_BASE_CHAIN>
 
+chains: '31337:http://localhost:8545,31338:http://localhost:8546,84532:https://sepolia.base.org,11155111:https://ethereum-sepolia-rpc.publicnode.com'
```

Comment:
This is needed for local and CI testing to be successful

Reply:
what is the reason for using external blockchains for local tests?

Reply:
anvil doesn't behave the same way as external chains, which has exposed us to issues in the past because xchain incompatibilities weren't caught until deploying. However I'm not sure we're testing external chains on stream node, this is just the same config as what xchain is using for local testing, which I think makes sense - we still have two node config template files at this time, maybe worth refactoring.